packages/arb-token-bridge-ui/src/hooks/TransferPanel/useIsBatchTransferSupported.ts

```diff
@@ -30,6 +33,10 @@ export const useIsBatchTransferSupported = () => {
     return false
   }
 
+  if (isLifi && !isArbitrumCanonicalTransfer) {
```

Comment:
Again, can `isArbitrumCanonicalTransfer` have `!isLifi` within it?

Reply:
See https://github.com/OffchainLabs/arbitrum-token-bridge/pull/2534#discussion_r2237022669